### DISCUSSION TITLE
Access from local network?

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [hlky/stable-diffusion-webui#438] [Q&A]
I want to access the webui from a different PC, but i get "Invalid HTTP request received."
http://localhost:7860/ on the same PC works fine.

**Comment by inukai132** [comment]
Check your firewall settings. I am able to access mine remotely so I think your issue might be a configuration issue

**Reply by KyuubiYoru** [reply]
I don't think the firewall blocks it, otherwise i would't get "Invalid HTTP request received."
stable-diffusion-ui worked right away, i think the server needs an argument that is not exposed.

**Comment by codedealer** [comment]
How do you access it? From another device in the same network you must access it via your machine's IP address.

E.g. `http://192.168.1.200:7860/`

**Reply by KyuubiYoru** [reply]
Found the problem, my browser defaulted to https instead of http.
